templates/stitch_images.ipynb

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
<p>If the <code>extraction_dir</code> doesn't exist this will fail. </p><p>We should make a separate <code>stitched_dir</code> as the output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>just realized that os.makedirs() call is unnecessary because the extraction_dir needs to exist for the function to load the tiffs from.  currently the stitched images get stored in a subdir <code>D:\\Extracted_Images\\stitched_images</code> . </p><p> </p><p>also is this going to get a 3e tag and we can assume the user has already extracted the images? or should I add some code to call bin_extraction.extract_missing_fovs()?</p>